pkg/asset/templates/kube-apiserver.yaml

```diff
@@ -17,8 +17,13 @@ spec:
         master: "true"
       hostNetwork: true
       containers:
+      - name: checkpoint-installer
+        image: quay.io/derek_parker/checkpoint-installer:v1.0.0
+        volumeMounts:
+        - mountPath: /etc/kubernetes/manifests
+          name: etc-k8s-manifests
       - name: kube-apiserver
-        image: quay.io/coreos/hyperkube:v1.3.0-alpha.5_coreos.0
+        image: quay.io/peanutbutter/hyperkube:v1.3.0-alpha.5_derek.1
```

Comment:
I'm fine with updating these for consistency, but wouldn't we only need the patched code in the kubelets?